backend/src/test/java/heartbeat/controller/report/GenerateReporterControllerTest.java

```diff
@@ -160,18 +144,90 @@ void shouldReturnWhenExportCsv() throws Exception {
 	}
 
 	@Test
-	void shouldGetExceptionAndPutInExceptionMap() throws Exception {
+	void shouldReturnAcceptedStatusAndCallbackUrlAndIntervalWhenCallBoardReports() throws Exception {
+		ReportResponse expectedResponse = ReportResponse.builder()
+			.velocity(Velocity.builder().velocityForSP(10).build())
+			.deploymentFrequency(DeploymentFrequency.builder()
+				.avgDeploymentFrequency(new AvgDeploymentFrequency("Average", 0.10F))
+				.build())
+			.build();
+
 		ObjectMapper mapper = new ObjectMapper();
-		GenerateReportRequest request = mapper
-			.readValue(new File("src/test/java/heartbeat/controller/report/request.json"), GenerateReportRequest.class);
+		GenerateBoardReportRequest request = mapper.readValue(
+				new File("src/test/java/heartbeat/controller/report/request.json"), GenerateBoardReportRequest.class);
```

Comment:
1.extract variable: request.json file path and csvtime stamp
2.remove given when then comments in unit test

Reply:
done